.travis.yml

```diff
@@ -181,18 +187,38 @@ jobs:
       - bin/release_docker $TRAVIS_TAG
       - if [[ $TRAVIS_TAG != *-rc* ]] && [[ $TRAVIS_TAG != *-dev ]]; then bin/release_docker latest; fi
       name: "Pushing release to hub.docker.com"
+
     - script:
       - bin/s3 sync s3://travis-$TRAVIS_BUILD_NUMBER/build-artifacts build/package
       - openssl aes-256-cbc -K $encrypted_2698a83519b3_key -iv $encrypted_2698a83519b3_iv -in bin/package/android/key.gpg.enc -out bin/package/android/key.gpg -d
       - gpg --import bin/package/android/key.gpg
-      - gpg --list-keys
       - bin/release_android "$TRAVIS_TAG"
       name: "Publish Android SDK to the Sonatype and Maven Central"
+
     - script:
       - bin/travis_scripts/trigger_desktop_app_build
       name: "Trigger Desktop APP build"
       if: tag = env(BUILD_DEV_RELEASE)
 
+    - script:
+      - openssl aes-256-cbc -K $encrypted_2698a83519b3_key -iv $encrypted_2698a83519b3_iv -in bin/package/android/key.gpg.enc -out bin/package/android/key.gpg -d
+      - gpg --import bin/package/android/key.gpg
+      - openssl aes-256-cbc -K $encrypted_2c668321e34e_key -iv $encrypted_2c668321e34e_iv -in bin/package/ppa/ppa_key.enc -out ~/.ssh/id_rsa -d
+      - chmod 600 ~/.ssh/id_rsa
+      - bin/release_ppa "$PPA_VERSION" "$BUILD_NUMBER" xenial
+      - bin/release_ppa "$PPA_VERSION" "$BUILD_NUMBER" bionic
```

Comment:
Why are we releasing just for ubuntu? IMHO we can release for debian too: `https://launchpad.net/debian/stretch`

Reply:
I don't think we can build PPA for Debian:
![image](https://user-images.githubusercontent.com/8612618/57906145-d6780f80-789a-11e9-9adc-2297396f993d.png)